install/kubernetes/helm/istio-remote/templates/sidecar-injector-configmap.yaml

```diff
@@ -170,7 +170,7 @@ data:
             - NET_ADMIN
           runAsGroup: 1337
           {{ "[[ else -]]" }}
-          {{ if and .Values.global.sds.enabled .Values.global.sds.enableTokenMount }}
+          {{ if and .Values.global.sds.enabled .Values.global.sds.useK8sSATrustworthyJwt }}
```

Comment:
For a better UX, consider using a shorter and less typo-prone name, like useTrustworthyJwt, or useJwt (if all Jwt are trustworty).

Reply:
renamed.